ogr_fdw_info.c

```diff
@@ -84,18 +96,26 @@ static void
 usage()
 {
 	printf(
-	    "usage: ogr_fdw_info -s <ogr datasource> -l <ogr layer>\n"
-	    "	   ogr_fdw_info -s <ogr datasource>\n"
-	    "	   ogr_fdw_info -f\n"
-	    "\n");
+		"usage: ogr_fdw_info -s <ogr datasource> -l <ogr layer name> -i <ogr layer index (numeric)> -t <output table name> -n <output server name> -o <config options>\n"
+		"       ogr_fdw_info -s <ogr datasource>\n"
+		"usage: ogr_fdw_info -f\n"
+		"       Show what input file formats are supported.\n"
+		"\n");
+	printf(
+		"note (1): You can specify either -l (layer name) or -i (layer index) if you specify both -l will be used\n"
```

Comment:
Just for my ignorance and laziness, when I get a list of layers back, does it include the layer index number as well as the layer name? (And for my curiosity, why can you not reference sheets by name?)

Reply:
Our automated process takes client supplied Excel spreadsheets and loads the first worksheet (worksheet index 1). Excel allows for worksheets to be called anything i.e. not just Sheet1 so if I was going to use sheet name I would first have to parse out sheet names and then load the first.